src/site/content/en/blog/http-cache-security/index.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Improve security and privacy by updating HTTP Cache
+authors:
+  - arthursonzogni
+description: Forgetting or misusing the Cache-Control header might negatively impact the security of your website and your users' privacy. Get recommendations for high-value websites.
+date: 2022-02-07
+thumbnail: image/VbsHyyQopiec0718rMq2kTE1hke2/3MnHBfutFgu2ai1F2mUH.jpg
+alt: Two ducks move through the water, creating a rippling wave.
+tags:
+  - blog
+  - security
+---
+
+By default, resources are always allowed to be cached by any type of cache.
+Not using or misusing the `Cache-Control` header might negatively impact the
+security of your website and your users' privacy.
+
+For personalized responses you want to keep private, we recommend you either:
+
+*  Prevent intermediaries from caching the resource. Set
+   `Cache-Control: private`.
+*  Set an appropriate [secondary cache
+   key](https://httpwg.org/specs/rfc7234.html#caching.negotiated.responses).
+   If the response varies due to cookies&mdash;which can happen when the
+   cookie stores credentials&mdash;set `Vary: Cookie`.
+
+Read on to learn why this matters and discover:
+
+1. Security and privacy problems you might be unaware of
+2. Different types of HTTP caches and common misconceptions
+3. Recommended actions for high-value websites
+
+## Cache-related security and privacy risks
+
+### Leaky resources from Spectre vulnerabilities
+
+The [Spectre
+vulnerability](https://en.wikipedia.org/wiki/Spectre_(security_vulnerability))
+allows a page to read an OS process's memory. This means an attacker can
+gain unauthorized access to cross-origin data. As a consequence, modern web
+browsers have restricted usage of some of their features&mdash;such as
+[`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+or [high resolution timer](https://developer.chrome.com/blog/cross-origin-isolated-hr-timers/)&mdash;to
+pages with [cross-origin
+isolation](/cross-origin-isolation-guide/). 
+
+Modern web browsers enforce [Cross-Origin Embedder Policy
+(COEP)](https://developer.chrome.com/blog/coep-credentialless-origin-trial/). This ensures cross-origin
+resources are either:
+
+*  Public resources, requested without cookies
+*  Resources explicitly allowed to be shared cross-origin, via CORS or the
+   [CORP](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+   header
+
+The COEP setup doesn't prevent an attacker from exploiting Spectre. However, it
+ensures cross-origin resources are not valuable to the attacker (when shared
```

Comment:
In the first case, the action is made by the browser, by not sending cookies. So I would rather say "when loaded as").
In the second case, the action is made by the server, so I would keep "when shared with"

Reply:
Done!